src/Points/expensePoints.js

```diff
@@ -0,0 +1,152 @@
+'use strict';
+
+const { getAllExpenses,
+  getExpenseById,
+  updateExpense,
+  createExpense,
+  deleteExpense } = require('../Postgre/postgreExpense');
```

Comment:
```suggestion
const { 
  getAllExpenses,
  getExpenseById,
  updateExpense,
  createExpense,
  deleteExpense, 
} = require('../Postgre/postgreExpense');
```
maybe it will be easier to read